shell/platform/darwin/ios/ios_context.h

```diff
@@ -45,10 +45,17 @@ class IOSContext {
   ///
   /// @param[in]  api       A client rendering API supported by the
   ///                       engine/platform.
+  /// @param[in]  backend   A client rendering backend supported by the
+  ///                       engine/platform.
+  /// @param[in]  msaa_samples
+  ///                       The number of MSAA samples to use. Only supplied to
+  ///                       Skia, must be either 0, 1, 2, 4, or 8.
```

Comment:
What happens when an invalid value is passed. Do we default to certain value if an invalid value is passed, a crash or there will be undefined behavior.

Are we preventing it on our side or the Skia API has something to prevent it.